CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * [#134](https://github.com/alexa-js/alexa-app/issues/134): Asynchronous support purely through Promises and removing `return false`/callback functionality - [@ajcrites](https://github.com/ajcrites).
 * [#22](https://github.com/alexa-js/alexa-app/issues/22): Asynchronous support for `pre` and `post` - [@ajcrites](https://github.com/ajcrites).
 * [#188](https://github.com/alexa-js/alexa-app/issues/188): Use `callback` to complete lambda functions rather than `context`. - [@ajcrites](https://github.com/ajcrites).
+* [#199](https://github.com/alexa-js/alexa-app/issues/199) Add support for `PlaybackController` events
```

Comment:
Make this look like the other lines please, add `- [@tternes](...).`, add a colon after the issue number.